data/json/items/tools.json

```diff
@@ -257,6 +257,29 @@
   },
 
   {
+    "id": "tailors_kit",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "red",
+    "name": "tailor's kit",
+    "description": "This is a high quality, plastic kit with a variety of needles, some plastic spools for thread, and a few other useful textile tools.  Use a tailor's kit to customize your clothing and armor.  This uses your tailoring skill.",
+    "price": 1000,
+    "material": ["plastic", "steel"],
+    "weight": 100,
+    "volume": 2,
+    "bashing": -1,
+    "cutting": 0,
+    "to_hit": -2,
+    "max_charges": 400,
+    "initial_charges": 50,
+    "charges_per_use": 1,
+    "turns_per_charge": 0,
+    "ammo": "thread",
+    "revert_to": "null",
+    "use_action": "SEW_ADVANCED"
```

Comment:
Looking below it, the tailor kit could include a mini scissors (include the scissor action & CUT 1), plenty of pins, a seam ripper, etc.  Currently it's a double-capacity sewing kit with the custom-modding menu, so requiring it should probably give some idea of what it contains above/beyond the standard kit.